docs/next-sitemap.config.js

```diff
@@ -1,5 +0,0 @@
-/** @type {import("next-sitemap").IConfig} */
```

Comment:
We should probably robots.txt ignore this as well, or whatever the hack we did was for preview builds